### ISSUE TITLE
Add support for the placeholder attribute

### ISSUE THREAD

**Issue by raffomania** [nakaibuild/nakai#14]
I'd like to add the `placeholder` attribute to a text input, I'd expect  to be able to do it like this:

```gleam
import nakai/html
import nakai/html/attrs

html.input([attrs.type_("text"), attrs.placeholder("Mochi")])
```

But the `placeholder` function seems to be missing in the `attrs` module. Is  this by design?

**Comment by aslilac** [comment]
Not intentional, just needs to be added to codegen/attrs.json.

**Comment by raffomania** [comment]
Would you like a MR for that? Seems like a simple change :)

**Comment by aslilac** [comment]
Yeah, always happy to accept contributions!

**Comment by aslilac** [comment]
thanks again for the fix! it's now published as part of v0.7
